cristin-person/src/test/java/no/unit/nva/cristin/person/handler/personAuthorized.feature

```diff
@@ -0,0 +1,94 @@
+Feature: API tests for Cristin Person fetch
+
+  Background:
+    * def domainName = java.lang.System.getenv('DOMAIN_NAME')
+    * def basePath = java.lang.System.getenv('BASE_PATH')
+    * def CRISTIN_BASE =  'https://' + domainName +'/' + basePath
+    * def illegalIdentifier = 'illegalIdentifier'
+    * def samplePersonId = '07117631634'
+    * def nonExistingPersonId = '11077941012'
+    * def personIdRegex = 'https:\/\/[^\/]+\/[^\/]+\/person\/[0-9]+'
+    * def PROBLEM_JSON_MEDIA_TYPE = 'application/problem+json'
+    * def token = 'just-a-invalid-token-for-now'
+    Given url CRISTIN_BASE
+    * print 'Current base url: ' + CRISTIN_BASE
+
+#  Scenario Outline: Fetch returns valid data and with correct content negotiation <CONTENT_TYPE>
+#    * configure headers = { 'Accept': <CONTENT_TYPE> }
+#    Given path '/person/identityNumber'
+#    * header Authorization = 'Bearer ' + token
+#    And request { type : NationalIdentificationNumber, value : '07117631634' }
+#    When method POST
+#    Then status 200
+#    And match response == '#object'
+#    And match response['@context'] == '#present'
+#    And match response.id == '#regex ' + personIdRegex
+#    And match response.type == 'Person'
+#
+#    Examples:
+#      | CONTENT_TYPE       |
+#      # TODO: Implement ld json
+#      #| 'application/ld+json' |
+#      | 'application/json' |
+
+
+  Scenario: Fetch returns status 401 Forbidden when requesting NationalIdentificationNumber without valid token
```

Comment:
Have you tried without token at all? Sometimes it only gives 4xx when supplied with some token that is invalid meaning not supplying a token at all can give 200